doc/cephadm/services/rgw.rst

```diff
@@ -303,6 +304,13 @@ where the properties of this service specification are:
 * ``ssl_cert``:
     SSL certificate, if SSL is to be enabled. This must contain the both the certificate and
     private key blocks in .pem format.
+* ``use_keepalived_unicast``
```

Comment:
sorry, not super familiar with keepalived internals so maybe a simple question, but what was the actual behavior prior to this change with both the interface and unicast IPs specified? Reading through the description of this new parameter plus the PR desctiption, it seems like it was either using the unicast IPs but would just limit the IPs it was willing to communicate with to those within the interface, or it was using multicast and the IPs were not being properly used. Are either of those correct? Just trying to fully understand the actual behavior change here.

Reply:
@adk3798 unicast. See my comment: https://github.com/ceph/ceph/pull/52020#issuecomment-1593145171